go_covert_lib/client/src/App.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react';
+import Button from 'react-bootstrap/Button';
+import FormControl from 'react-bootstrap/FormControl';
+import InputGroup from 'react-bootstrap/InputGroup';
+import './styles.css';
+
+/**
+ * IMPORTANT NOTE: For styling, refer to https://getbootstrap.com/docs/4.0/utilities/position/
+ */
+
+const App = () => {
+  const [textToSend, setTextToSend] = useState('');
+  const [channels, setChannels] = useState([]);
+  let ws; // The websocket connection to the server
+
+  const sendConfig = () => {
+    const cmd = JSON.stringify({ OpCode: 'config' });
+    ws.send(cmd);
+  };
+
+  useEffect(() => {
+    // Matches just the "127.0.0.1:8080" portion of the address
+    const addressRegex = /[a-zA-Z0-9.]+:[\d]+/g;
```

Comment:
If the regex does not match the regex an error will occur when you try to index. Is this what you want? Or do you want more sophisticated error handling, like a message to the user.
It will also fail if the user has  connected to "localhost:8080".

Reply:
Yeah sure, I can fix up the error handling.

The regex matches on alphanumeric characters as well, so `localhost:8080` will be matched.

Reply:
Fixing this in #22 , as I needed to tweak/move the websocket code in that branch